build-system/tasks/karma.conf.js

```diff
@@ -18,7 +18,7 @@
 /**
  * @param {!Object} config
  */
-module.exports = {
+module.exports = {  // eslint-disable-line no-undef
```

Comment:
We should set the `eslintrc` to `node` environment.

Reply:
Done.